psi4/driver/procrouting/proc.py

```diff
@@ -1250,7 +1250,9 @@ def scf_helper(name, **kwargs):
     scf_wfn = scf_wavefunction_factory(core.get_option('SCF', 'REFERENCE'), base_wfn)
     core.set_legacy_wavefunction(scf_wfn)
 
-    read_filename = core.get_writer_file_prefix(scf_molecule.name()) + ".180.npz"
+    fname = core.get_writer_file_prefix(scf_molecule.name()) + ".180.npz"
+    read_filename = os.path.join(core.get_environment("PSI_SCRATCH"), fname)
```

Comment:
Seems more understandable I think. Also, PSIO is probably going to be deprecated sooner or later.